tests/test_extended.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import numpy as np
+import tensorflow as tf
+
+import zfit
+from zfit.core.sample import extract_extended_pdfs, extended_sampling
+
+obs1 = 'obs1'
+
+
+@pytest.mark.flaky(reruns=5)
```

Comment:
Why flaky?

Reply:
Because of the sampling, how should we verify that it sampled poissonian? but 3 reruns is ok as well.

It's hard in the end to test if the sampling is correct, we just check the sum to be roughly the same now